src/main/java/org/creekservice/kafka/test/perf/implementations/SkemaImplementation.java

```diff
@@ -71,22 +77,32 @@ public JsonValidator prepare(
             final AdditionalSchemas additionalSchemas,
             final boolean enableFormatAssertions) {
 
-        /*
-        Waiting on https://github.com/erosb/json-sKema/commit/ee8aca8c1452688dba485fa7cc7f3ab4fc85d74c being released
-        before we can make use of enableFormatAssertions
-         */
-
         final JsonValue schemaJson = new JsonParser(schema).parse();
 
+        final SchemaClient schemaClient =
+                new SchemaClient() {
+                    @Override
+                    public InputStream get(final URI uri) {
+                        return new ByteArrayInputStream(
+                                additionalSchemas.load(uri).getBytes(UTF_8));
+                    }
+
+                    @Override
+                    public IJsonValue getParsed(final URI uri) {
+                        return SchemaClient.DefaultImpls.getParsed(this, uri);
+                    }
```

Comment:
Unfortunately, default method impls on Kotlin interfaces don't play nicely with Java... :(

Reply:
That's very sad, didn't know that.
By the way I also added support in 0.11.0 to pre-register schemas before loading. Maybe that would sufficient here, instead of having a custom SchemaClient. [Example](https://github.com/erosb/json-sKema-examples/blob/master/src/main/java/com/github/erosb/jsonsKema/examples/PreRegisteredSchemas.java#L20)

Reply:
Nice. And Done.